src/main/scss/base/_style.scss

```diff
@@ -389,6 +391,24 @@ pre.console {
   p:last-of-type {
     margin-bottom: 0;
   }
+
+  // add spacing above headings except for when its the first element in the help
+  // the need for this is caused by p:last-of-type setting margin-bottom to 0
+  // unfortunately because of the varied markup I wasn't able to find a way to avoid this
+  h1:not(:first-child),
```

Comment:
@janfaracik what do you think about this approach and can you think of a simpler one?

Reply:
Yeah I can't think of anything sadly, could use 'gap' on the parent container but due to how varied the contents are it probably would make things worse :/